docs/keyboard.md

```diff
@@ -19,7 +19,7 @@ class Example extends Component {
     this.keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', this._keyboardDidHide);
   }
 
-  componentWillUnmount () {
```

Comment:
You're right about `componentWillMount` documentation, but here we have `componentWillUnmount`, which is the right place to remove listeners.